rename_wpt/main.go

```diff
@@ -13,9 +15,24 @@ import (
 // appropriately in the wpt-metadata repository. This program will roll in the latest WPT SHA once a day
 // through a cron job.
 func main() {
-	// TODO(Kyleju): plumb the before, after SHA through shell. Update SHA through shell as well.
-	shaAfter := "e021a4c8f92596f3261e2d7540d017f5138deb68"
-	shaBefore := "012651bc39bb45a6dc9c93a968b47dd9310a1e8a"
+	shaAfter := updateManifest()
+	if shaAfter == "" {
+		fmt.Println("Unable to update manifest and get shaAfter")
+		return
```

Comment:
If we reach this code path, will the exit code be zero or non-zero? It's important for failures to fail in CI, and my guess is this that this will silently print to stdout and the appear as a success.